src/lib/command/util/util-util.ts

```diff
@@ -0,0 +1,34 @@
+import { SmartThingsClient } from '@smartthings/core-sdk'
+
+import { APICommand } from '../api-command.js'
+import { ChooseOptions, chooseOptionsWithDefaults, stringTranslateToId } from '../command-util.js'
+import { SelectFromListConfig, SelectFromListFlags, selectFromList } from '../select.js'
+
+
+export type ChooseFunction<T extends object> = (
+	command: APICommand<SelectFromListFlags>,
+	locationFromArg?: string,
+	options?: Partial<ChooseOptions<T>>) => Promise<string>
+
+export const createChooseFn = <T extends object>(
+	config: SelectFromListConfig<T>,
+	listItems: (client: SmartThingsClient) => Promise<T[]>,
+): ChooseFunction<T> =>
+	async (
+			command: APICommand<SelectFromListFlags>,
+			locationFromArg?: string,
+			options?: Partial<ChooseOptions<T>>): Promise<string> => {
+		const opts = chooseOptionsWithDefaults(options)
+
+		const listItems2 = (): Promise<T[]> => listItems(command.client)
```

Comment:
Oh, shoot. I didn't mean to leave that. I gave it that name because I didn't want to break my stride and I had planned to go back and fix it. It's fixed now.

BTW, I like `Wrapper` too. I don't think I would have come up with that on my own. I probably would have done something like named the argument `listItemsIn` and given this variable `listItems` but I like the `Wrapper` suffix better so I'm kinda glad I forgot. 😄